script/deploy/healthCheck.mts

```diff
@@ -109,7 +109,7 @@ const main = defineCommand({
       `../../script/deploy/_targetState.json`
     )
     const nonCoreFacets = Object.keys(
-      targetStateJson[network.toLowerCase()].production.LiFiDiamond
+      targetStateJson[network.toLowerCase()].production.LiFiDiamond,
```

Comment:
why is it adding a , at the end?

Reply:
This is pretty common. It's mainly used to be able to add new items later without having to remember to add another comma in the line above.